bifrost/release.go

```diff
@@ -250,6 +250,10 @@ func (r *Release) GrabLocks(s3c aws.S3API) error {
 		return err
 	}
 
+	if err := r.GrabUserLock(s3c); err != nil {
```

Comment:
I think this will create a UserLock which we don't want. What we need is it just to check if it is there, the user has to actually grab and release this particular lock

Reply:
If it is there the error should contain the user who locked it though, and their reason

Reply:
Ohh I was originally thinking that we wouldn't allow users to lock a configuration if there was currently an outgoing deploy, but I guess that doesn't really matter.